### ISSUE TITLE
Error when I try to add a sensor 

### ISSUE THREAD

**Issue by BragaGuilherme** [lordmilko/PrtgAPI#365]
### What's going on?

Hello Lord Milko, how are you?

I was trying to run this command: 

$table = @{ "name_" = "PCT_CROMATOGRAFIA_NITROGENIO"; "pythonscript_" = "Modbus Python.py";"params_" ="registro=17231,nome=PCT_CROMATOGRAFIA_NITROGENIO,tipo=32float";"sensortype" = "pythonscript";"comments_"="teste"};$params = New-SensorParameters $table; Get-Device -Id 39273 | Add-sensor $params;

 It was working fine earlier, but now it keeps giving me this error: 

Add-sensor : Could not authenticate to PRTG; the specified username and password were invalid.
No linha:1 caractere:280
+ ... w-SensorParameters $table; Get-Device -Id 39273 | Add-sensor $params;
+                                                       ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-Sensor], HttpRequestException
    + FullyQualifiedErrorId : System.Net.Http.HttpRequestException,PrtgAPI.PowerShell.Cmdlets.AddSensor

Can you help me with this?

<img width="863" alt="Captura de tela 2023-11-06 155200" src="https://github.com/lordmilko/PrtgAPI/assets/111820979/fa284932-d601-44e2-8c4a-cd5955af1a9f">

### Due Dilligance

- [X] It wasn't covered by the [wiki](https://github.com/lordmilko/PrtgAPI/wiki), I swear!
- [X] I have tried doing some basic research on this issue but haven't been able to come up with anything. Please help!

**Comment by lordmilko** [comment]
Please provide the output of `Get-PrtgClient -Diagnostic`

**Comment by BragaGuilherme** [comment]
Here lord,
<img width="392" alt="image" src="https://github.com/lordmilko/PrtgAPI/assets/111820979/a9bcd206-3a00-4e4d-90bf-0db7dcae1aa2">

**Comment by xasz** [comment]
What Sensor are you trying to add ? 
I have never added a phytonscript sensor, but i think here are some basic problems.
At least on my instance the `New-SensorParameters` Cmdlet have not -Type phytonscript

Furthermore if you want to build hashmap for the parameters, which i see no point in that szenario, you need to use `@`
like `New-SensorParameters @table`



@BragaGuilherme This should be possible with basic powershell piping: 
```powershell
Get-Device -Id 148376 | Get-Sensor -Name Ping | Pause-Object -Forever -Message "You are not needed"
```

**Comment by BragaGuilherme** [comment]
> Here lord, <img alt="image" width="392" src="https://user-images.githubusercontent.com/111820979/281094160-a9bcd206-3a00-4e4d-90bf-0db7dcae1aa2.png">

This issue has reappeared. What could I do to work around it?

**Comment by xasz** [comment]
What issue? You should add sime information... ?

**Comment by BragaGuilherme** [comment]
[imagem]

**Comment by nickhoksbergen** [comment]
I have the same issue when adding sensors with either the New-Sensor or Add-Sensor cmdlets.

**This is the code I run:**

$device = $newgroup | get-device |  select -first 1 
$device | new-sensor -ExeFile "NonDefaultServicesV2.ps1" -Name "Non Default Services" -ExeXml

$params = New-SensorParameters ExeXml "Non Default Services"
$params.ExeFile = "NonDefaultServicesV2.ps1"
$params.ExeParameters = "-FQDN %host"
$device | Add-Sensor $params

**This is the result**

Add-Sensor : Could not authenticate to PRTG; the specified username and password were invalid.
At line:4 char:11
+ $device | Add-Sensor $params
+           ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-Sensor], HttpRequestException
    + FullyQualifiedErrorId : System.Net.Http.HttpRequestException,PrtgAPI.PowerShell.Cmdlets.AddSensor

**Result of 'Get-PrtgClient -Diagnostic'**
PSVersion      : 5.1.22621.2428
PSEdition      : Desktop
OS             : Microsoft Windows 11 Enterprise
PrtgAPIVersion : 0.9.18
Culture        : nl-NL
CLRVersion     : 533320
PrtgVersion    : 23.4.88.1429
PrtgLanguage   : english.lng

**Comment by xasz** [comment]
Your prtgapi Version ist Not Up2date.

**Comment by BragaGuilherme** [comment]
> What issue? You should add sime information... ?

Yes, I am trying to add a PythonScript sensor, which has worked before. The issue is that when I try to execute it, accessing another PRTG that is only accessible via IP and does not have a security certificate, this happens.

**Comment by xasz** [comment]
Does IT Work any other Sensors. I think you should Test one oneliner on which command IT Is failing. I cannot See that from you informations posted.

**Comment by lordmilko** [comment]
I tested PrtgAPI 0.9.19 on PRTG 23.4.90.1299+

```powershell
$params = New-SensorParameters ExeXml "Non Default Services"
$params.ExeFile = "NonDefaultServicesV2.ps1"
$params.ExeParameters = "-FQDN %host"
$device | Add-Sensor $params
```
I was able to run this code

Are you able to update to PRTG 23.4.90.1299+ and advise if you're still having the issue? I didn't test on an earlier version of PRTG

If you're still having issues are you potentially able to try a ping sensor as well?

```powershell
$device | New-SensorParameters -rt ping | Add-Sensor
```
